src/objects/Getting_Started_Completed_Checklist__c.object

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <actionOverrides>
+        <actionName>Accept</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>CancelEdit</actionName>
```

Comment:
Is this a typo? If it's intentional, you can ignore this comment.

Reply:
This is not a typo, its metadata format